install.sh

```diff
@@ -4,6 +4,7 @@ set -e
 
 BATS_ROOT="${0%/*}"
 PREFIX="$1"
+BASELIB="${2:-lib}"
```

Comment:
If we only want to support lib and lib64, shouldn't we just check the existence of lib vs lib64 and choose automatically?

Reply:
For cross compile, it can not check the directory on build machine and requires to pass the libdir to the script. BTW, this patch breaks some existing script and I'll send v2. Thanks.

Reply:
Do you have an ETA for that?

Reply:
It turns out the failure I mean is not bats related, so this version of the patch is enough. But I am not familiar with writing bats test cases. Is there any doc about it, please? Thanks.

Reply:
I think you can get inspiration from the other tests in test/install.bats. Aditionally, you can view the normal documentation on readthedocs.

Reply:
@martin-schulze-vireso I copied and updated a test case in install.bats. Would you like to have a look at it, please? Thanks.